arrow/src/main/java/org/apache/calcite/adapter/arrow/ArrowTable.java

```diff
@@ -194,6 +194,8 @@ private static TreeNode makeLiteralNode(String literal, String type) {
       return TreeBuilder.makeLiteral(parseFloat(literal));
     case "double":
       return TreeBuilder.makeLiteral(parseDouble(literal));
+    case "decimal":
+      return TreeBuilder.makeDecimalLiteral(literal, 19, 0);
```

Comment:
Comply with Calcite's default Decimal type

Reply:
You should use the type system to retrieve the default type, assuming it is available here.
Maybe there is a default type for Arrow, but the "default" type for Calcite is irrelevant in this context.

Reply:
There is no way this can be right, this drops all digits after the decimal point.
At least make 19 and 0 some constants like ARROW_DEFAULT_PRECISION and ARROW_DEFAULT_SCALE.

Reply:
done